src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FileRouteSpec.scala

```diff
@@ -23,7 +23,6 @@ class FileRouteSpec extends TestContainerUtils with Matchers with TestRequest {
   case class FileNameAndPath(fileName: String, path: String)
```

Comment:
There are a couple of case classes that refer to "all descendants" that should be removed:
- line 30: `case class GraphqlQueryDataAllDescendants`;
- line 37: `case class AllDescendants`